packages/bpk-component-price-range/src/BpkPriceRange.module.scss

```diff
@@ -0,0 +1,134 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:math';
+@use '../../unstable__bpk-mixins/tokens';
+@use '../../unstable__bpk-mixins/utils';
+
+$small-lines-height: tokens.bpk-spacing-sm();
+$large-lines-height: tokens.bpk-spacing-md();
+
+.bpk-price-range {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  padding-block: tokens.bpk-spacing-sm();
+  row-gap: tokens.bpk-spacing-sm();
+
+  &--large {
+    padding-block: 0;
+  }
+
+  &__lines {
+    position: relative;
+    display: flex;
+    height: $small-lines-height;
+    flex: none;
+    gap: math.div(tokens.bpk-spacing-sm(), 2);
+
+    &--large {
+      height: $large-lines-height;
+    }
+  }
+
+  &__line {
+    &--low {
+      width: calc(100% * var(--low));
+      height: 100%;
+      background-color: tokens.$bpk-status-success-spot-day;
+      border-end-start-radius: tokens.$bpk-border-radius-sm;
+      border-start-start-radius: tokens.$bpk-border-radius-sm;
+
+      &Large {
+        border-end-start-radius: tokens.$bpk-border-radius-md;
+        border-start-start-radius: tokens.$bpk-border-radius-md;
+      }
+    }
+
+    &--typical {
+      width: calc(100% * (var(--high) - var(--low)));
+      height: 100%;
+      background-color: tokens.$bpk-core-primary-day;
+    }
+
+    &--high {
+      width: calc(100% * (1 - var(--high)));
+      height: 100%;
+      background-color: tokens.$bpk-text-error-day;
+      border-end-end-radius: tokens.$bpk-border-radius-sm;
+      border-start-end-radius: tokens.$bpk-border-radius-sm;
+
+      &Large {
+        border-end-end-radius: tokens.$bpk-border-radius-md;
+        border-start-end-radius: tokens.$bpk-border-radius-md;
+      }
+    }
+
+    &--dot {
+      position: absolute;
+      top: 50%;
+      display: flex;
+      width: $small-lines-height * 3;
+      height: $small-lines-height * 3;
+      justify-content: center;
+      align-items: center;
+      transform: translate(0, -50%);
+      border-radius: 50%;
+      margin-inline-start: var(--prefilled-width);
+
+      &::after {
+        content: '';
+        display: block;
+        width: $small-lines-height;
+        height: $small-lines-height;
+        border-radius: 50%;
+        background-color: tokens.$bpk-text-primary-inverse-day;
+      }
+    }
+  }
+
+  &__marker {
+    width: fit-content;
+    margin-inline-start: var(--prefilled-width);
+  }
+
+  &__ranges {
+    position: relative;
+    display: flex;
+    width: 100%;
+    height: tokens.$bpk-line-height-base-tight;
```

Comment:
Ideally we should use the text tokens to set the height of the container.

We can either:

- Let the text itself handle this 
- Just set a raw `rem` value here which is acceptable so it would be `1.25rem`